src/searchbar.cpp

```diff
@@ -63,16 +63,18 @@ SearchBar::SearchBar(QWidget *parent) :
     setPlaceholderText(tr("Search"));
     m_completer.setCompletionMode(QCompleter::UnfilteredPopupCompletion);
     setCompleter(&m_completer);
+    m_completer.setCaseSensitivity(Qt::CaseInsensitive);
```

Comment:
@mgautierfr 
I think that it's easier to set the QCompleter handle the focus on the right row setting the case insensitive instead of rewrite the focus of the different widgets.
Even if the app has a custom suggestion's list engine, say to the QCompleter to match with the case insensitive doesn't seem inconsistent to me